webapp/src/pages/Map.tsx

```diff
@@ -161,12 +161,21 @@ function MapPage(props: MapPageProps) {
   const [startDate, setStartDate] = useState(new Date(baseDate.getTime() + 60000 * filterOffset * filterStep))
   const [endDate, setEndDate] = useState(new Date(startDate.getTime() + 60000 * filterStep))
   const stepOptions = [
-    { value: 1, label: '1 minute', options: [] as number[] },
-    { value: 5, label: '5 minutes', options: [] as number[] },
-    { value: 15, label: '15 minutes', options: [] as number[] },
-    { value: 30, label: '30 minutes', options: [] as number[] },
-    { value: 60, label: '60 minutes', options: [] as number[] },
+    { value: 1, label: '1 minute' },
+    { value: 5, label: '5 minutes' },
+    { value: 15, label: '15 minutes' },
+    { value: 30, label: '30 minutes' },
+    { value: 60, label: '60 minutes' },
   ]
+  const [selectedOption, setSelectedOption] = useState({ value: 60, label: '60 minutes' })
```

Comment:
Are we using this value anywhere? I noticed that it's not passed to our Select component anywhere

Reply:
Good catch. This is leftover from a test that I needed to do to verify the issue with the selector display.